src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -46,81 +48,87 @@
 import android.os.Message;
 import android.os.Messenger;
 import android.os.RemoteException;
-import android.util.Log;
 
 /**
  * An class used to set up interaction with beacons from an <code>Activity</code> or <code>Service</code>.
  * This class is used in conjunction with <code>BeaconConsumer</code> interface, which provides a callback
  * when the <code>BeaconService</code> is ready to use.  Until this callback is made, ranging and monitoring
  * of beacons is not possible.
- * 
+ * <p/>
  * In the example below, an Activity implements the <code>BeaconConsumer</code> interface, binds
  * to the service, then when it gets the callback saying the service is ready, it starts ranging.
- * 
- *  <pre><code>
+ * <p/>
+ * <pre><code>
  *  public class RangingActivity extends Activity implements BeaconConsumer {
  *  	protected static final String TAG = "RangingActivity";
  *  	private BeaconManager beaconManager = BeaconManager.getInstanceForApplication(this);
- *  	 {@literal @}Override
+ *     {@literal @}Override
  *  	protected void onCreate(Bundle savedInstanceState) {
  *  		super.onCreate(savedInstanceState);
  *  		setContentView(R.layout.activity_ranging);
  *  		beaconManager.bind(this);
- *  	}
- *  	 {@literal @}Override 
+ *    }
+ *     {@literal @}Override
  *  	protected void onDestroy() {
  *  		super.onDestroy();
  *  		beaconManager.unbind(this);
- *  	}
- *  	 {@literal @}Override
+ *    }
+ *     {@literal @}Override
  *  	public void onBeaconServiceConnect() {
  *  		beaconManager.setRangeNotifier(new RangeNotifier() {
- *        	 {@literal @}Override 
+ *             {@literal @}Override
  *        	public void didRangeBeaconsInRegion(Collection<Beacon> beacons, Region region) {
  *     			if (beacons.size() > 0) {
- *	      			Log.i(TAG, "The first beacon I see is about "+beacons.iterator().next().getDistance()+" meters away.");		
- *     			}
- *        	}
- *  		});
- *  		
+ * 	      			Log.i(TAG, "The first beacon I see is about "+beacons.iterator().next().getDistance()+" meters away.");
+ *                }
+ *            }
+ *        });
+ * <p/>
```

Comment:
This <code>&lt;p/&gt;</code> breaks the layout in Android Studio. In a <code>&lt;pre&gt;</code> block a paragraph tag isn't needed to get line breaks.